src/resolvers/index.ts

```diff
@@ -158,12 +158,12 @@ const resolvers = {
             First_name,
             Last_Name,
             DOD_ID,
-            Grade,
```

Comment:
Note: I believe grade still comes back from the API too, but since it's repeated in Rank I too see no need to pull it back here.